examples/guided.py

```diff
@@ -388,5 +387,17 @@ def perform_installation(mountpoint):
 			except:
 				pass
 
-ask_user_questions()
+if not archinstall.arguments:
```

Comment:
Hmm, I feel this is too broad of a condition and might restrict what we can do too much. For instance, what if we want to do --config but have the user just provide passwords so they are not stored in a config file? Being able to selectively ask questions for just things not provided could be a valuable feature in the future.

Reply:
That should be the proper way to implement this, I was kind of going only for the basic ( silent ) install for now, so we could have declarative configs and everything else is automated... I might be able to modify this in the future to cover broader perspectives

Reply:
Would it be fine if the user sets sensitive info such as passwords through environment variables?

Reply:
Hmm, that could definitely be viable. It'd be strange to do as there is not a fixed number of users though. You can have as many as you want in ArchInstall.

Reply:
Can we have a .env file to pull users and passwords from, currently testing this locally and it seems to be working, but this will involve adding a requirement (**dotenv**)